src/main/java/core/basesyntax/bookstore/service/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.bookstore.service.impl;
+
+import core.basesyntax.bookstore.model.Book;
+import core.basesyntax.bookstore.repository.BookRepository;
+import core.basesyntax.bookstore.service.BookService;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+    private final BookRepository bookRepository;
+
+    public BookServiceImpl(BookRepository bookRepository) {
+        this.bookRepository = bookRepository;
+    }
```

Comment:
FYI you can use @RequiredArgsConstructor instead of this explicit constructor, it will work for final fields